sentry_sdk/client.py

```diff
@@ -750,6 +763,9 @@ def capture_event(
         if self.transport is None:
             return None
 
+        if is_error:
+            self._last_event_id = event_id
```

Comment:
Do we want `last_event_id` only to return the ID of the most recent error event? Or, should we expand it to all events? Error event seems like the better option to align with #3049

Reply:
Yes, I think only errors is fine. Because the error ID is important for user feedback form.